components/CustomerForm.tsx

```diff
@@ -83,17 +85,22 @@ const CustomerForm: React.FC<{
       value: CustomerType.SKIP_COMPLIANCE_AND_BANKING,
       group: '',
     },
-    { label: 'Skip Consumer Credit', value: CustomerType.SKIP_CONSUMER_CREDIT, group: '' },
```

Comment:
This is a super weird diff.

All I did here was comment out the last selectable choice (Skip Compliance + Banking + Consumer Credit).